packages/console-utils/xconsole-service/src/utils/index.ts

```diff
@@ -29,31 +30,36 @@ export const getConsoleConfig = (key) => {
   return config && config[key];
 };
 
-export const getRiskInfo = () => {
+export const getRiskInfo = (): any => {
   const riskInfo = getGlobalVariable(RISK_INFO);
   return riskInfo || {};
 };
 
-export const getSecToken = function() {
+export const getSecToken = (): any => {
   return getConsoleConfig(SEC_TOKEN);
 };
 
-export const getUmid = function() {
+export const getUmid = (): string => {
   const riskInfo = getRiskInfo();
   return riskInfo.UMID;
 };
 
-export const getCollina = function() {
+export const getCollina = (): string => {
   const riskInfo = getRiskInfo();
   if (typeof riskInfo.GETUA === 'function') {
     return riskInfo.GETUA() || 'Fake collina generated in [getCollina]';
   }
+  return '';
 };
 
-export const getLocale = function() {
+export const getLocale = (): string => {
   return getConsoleConfig(LOCALE);
 };
 
-export const getActiveRegionId = function() {
-  return Cookies.get(CURRENT_REGION_COOKIE_NAME) || Cookies.get(REGION_COOKIE_NAME) || 'cn-hangzhou';
+export const getActiveRegionId = (): string => {
+  return (
+    docCookies.getItem(CURRENT_REGION_COOKIE_NAME) ||
+    docCookies.getItem(REGION_COOKIE_NAME) ||
+    'cn-hangzhou'
```

Comment:
顺序先换一下。
```suggestion
docCookies.getItem(REGION_COOKIE_NAME) ||
docCookies.getItem(CURRENT_REGION_COOKIE_NAME) ||
'cn-hangzhou'
```
由于历史原因，控制台中 ActiveRegionId 的优先级应该是最高的，然后是 consolebase 的 currentRegionId. 这个逻辑开源版本之前就是错的。所以会出现问题 在 1.x 我顺序已经换了

Reply:
oki